### DISCUSSION TITLE
There is no public option

### DISCUSSION THREAD

**Discussion by mmuu1987** [pixiv/vroid-sdk-developers#76] [General]
Only YOURS LIKE PICKUP three options no public other get app I see have public this is where s wrong with me

**Comment by nyamadan** [comment]
We implement YOURS, LIKE, and PICKUP tabs fo the example app.

If you want to implement custom tabs, you can use [DefaultApi](https://developer.vroid.com/en/sdk/docs/0.1.1/Pixiv.VroidSdk.Api.DefaultApi.html) to load 3D models.
But, VRoid SDK does not allow you to specify detailed search conditions(e.g. the latest published 3D models others created).

**Comment by mmuu1987** [comment]
Thanks very much
